ga4gh/datamodel/datasets.py

```diff
@@ -213,15 +226,18 @@ def __init__(
             numAlignments=1, numFeatureSets=1):
         super(SimulatedDataset, self).__init__(localId)
         self._description = "Simulated dataset {}".format(localId)
+        sequenceOntology = ontologies.Ontology("sequence_ontology")
```

Comment:
I think there was a renaming `ontologies` -> `ontologyMaps` that didn't get captured here.